source/likely.js

```diff
@@ -4,6 +4,7 @@ var likely = require('./index.js');
 var history = require('./history');
 
 window.addEventListener('load', function () {
+    likely.defaults = likely.getDefaultOptions();
```

Comment:
Do you think we can use it on demand, just to call `getDefaultOptions()`? Or to drop it at all?